cinema/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Currently, there are no test cases implemented. Consider adding tests for the CRUD operations of the Movie model as specified in the task description. This will help ensure that your API endpoints are functioning correctly.